src/connectComp.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECTCOMP_H
+#define CONNECTCOMP_H
+
+#include <vector>
+#include <unordered_map>
+#include <string>
+
+namespace found {
+
+struct Point {
+    int x;
+    int y;
+};
+
+struct Edge {
+    int magnitude; // number of points in this edge
+    std::vector<Point> points;
+};
+
+// Function prototype
+std::vector<Edge> NaiveConnectedComponent(unsigned char *image, int imageWidth, int imageHeight);
+
+}
+
+#endif
```

Comment:
I know you have completed tests on this function, so I will need you to formalize that in the test suite. That can be found in "found/test/<nameOfThisFile>test.cpp" (No .hpp file needed here). `example.cpp` will help you figure out how to write tests using C++'s CHECK framework (I'm open to other test suites if its better by the way).